src/Orleans.Core/Configuration/Options/SiloStatisticsOptions.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+
+
+namespace Orleans.Hosting
+{
+    public class SiloStatisticsOptions : StatisticsOptions
+    {
+        public static readonly TimeSpan SILO_DEFAULT_PERF_COUNTERS_WRITE_PERIOD = TimeSpan.FromSeconds(30);
+
+        public SiloStatisticsOptions()
+        {
+            this.PerfCountersWriteInterval = SILO_DEFAULT_PERF_COUNTERS_WRITE_PERIOD;
+        }
+
+        public TimeSpan DeploymentLoadPublisherRefreshTime { get; set; } = DEFAULT_DEPLOYMENT_LOAD_PUBLISHER_REFRESH_TIME;
+        public static readonly TimeSpan DEFAULT_DEPLOYMENT_LOAD_PUBLISHER_REFRESH_TIME = TimeSpan.FromSeconds(1);
+
+
+        /// <summary>
+        /// The LoadShedding element specifies the gateway load shedding configuration for the node.
+        /// If it does not appear, gateway load shedding is disabled.
+        /// </summary>
+        public bool LoadSheddingEnabled { get; set; }
+
+        /// <summary>
+        /// The LoadLimit attribute specifies the system load, in CPU%, at which load begins to be shed.
+        /// Note that this value is in %, so valid values range from 1 to 100, and a reasonable value is
+        /// typically between 80 and 95.
+        /// This value is ignored if load shedding is disabled, which is the default.
+        /// If load shedding is enabled and this attribute does not appear, then the default limit is 95%.
+        /// </summary>
+        public int LoadSheddingLimit { get; set; } = DEFAULT_LOAD_SHEDDING_LIMIT;
```

Comment:
Agreed, but I was trying to avoid making too many tiny options objects, so I grouped things 'mostly' by the systems responsible for them.  The statistics system is currently responsible for this, even if it shouldn't be.  My preference would be to keep this here until we refactor the statistics system, but if you've a strong objection, I can move it to it's own options.

It should be noted that moving this to it's own options also means wiring up that new option in every place that needs this, most of which already had access to the statistics options.